lib/build.api.js

```diff
@@ -192,7 +192,8 @@ class Build {
             switch (link.conflict) {
               case 'overwrite':
                 this.aquifer.console.log('Destination exists. Conflict set to overwrite. \nOverwriting: ' + link.dest, 'status');
-                del.sync(link.dest);
+                options.force = true;
```

Comment:
Great catch.  At some point, I had tried to include this in aquifer.json file but that is not longer necessary.  I marked this as needs work and will update soon.